src/update_roster.R

```diff
@@ -345,17 +528,20 @@ build_rosters_weekly_ngsapi <- function(season) {
   progressr::with_progress({
     weekly_rosters <- scrape_rosters()
 
-    if(nrow(weekly_rosters) == 0) {
+    if (nrow(weekly_rosters) == 0) {
       # if season roster is missing, patch with current roster and leave weeks null
-      season_rosters <- ngsscrapR::scrape_current_roster(teamId = "ALL", season = season)
+      season_rosters <-
+        ngsscrapR::scrape_current_roster(teamId = "ALL", season = season)
       weekly_rosters <- weekly_rosters |>
         dplyr::bind_rows(season_rosters)
     }
 
     weekly_rosters <- weekly_rosters |>
       dplyr::group_by(gsis_id) |>
-      dplyr::mutate(entry_year = dplyr::first(na.omit(entry_year)),
-                    rookie_year = dplyr::first(na.omit(rookie_year))) |>
+      dplyr::mutate(
+        entry_year = dplyr::first(na.omit(entry_year)),
+        rookie_year = dplyr::first(na.omit(rookie_year))
```

Comment:
I think these firsts should eventually be switched to a mode e.g. nflfastR::custom_mode but I see this is just a reindenting change

Reply:
Yeah, and I think they're actually made redundant by the changes we make with `load_players()`. I think I'll change the logic around to calculate `years_exp` later.